ChangeLog.md

```diff
@@ -0,0 +1,10 @@
+# Changelog for snap
+1.0.1
+    - Added a package.yaml for the project.
+    - Untracked the .ghci file.
+    - Upgraded to lts-12.1
+    - Removed unnecesary comments on stack.yaml
+1.0
+    - Added a change log.
+    - Set up the initial folder structure for a snap project.
+    - Set up project structure for a 'stack' structure.
```

Comment:
I suppose you want to display a list here, but the generated markdown doesn't look quite right. Perhaps, you want to set the `1.0` and `1.0.1` like subtitles:
```markdown
## 1.0.1
- Added a ...
```

Reply:
@kinorius123 In addition to that. Why the version is `1.0.1` but the `package.yml` says `0.1.0.0`? Usually, you add things to your CHANGELOG accordingly with the version described in your cabal file (which is generated from `package.yml`).
Is that clear?